src/plots/ternary/layout_defaults.js

```diff
@@ -27,6 +27,7 @@ module.exports = function supplyLayoutDefaults(layoutIn, layoutOut, fullData) {
         attributes: layoutAttributes,
         handleDefaults: handleTernaryDefaults,
         font: layoutOut.font,
+        autotypenumbersDflt: layoutOut.autotypenumbers,
```

Comment:
unnecessary?

https://github.com/plotly/plotly.js/blob/4b5bfcc73782f935210db720fc52accde634869e/src/plots/ternary/layout_defaults.js#L78

Reply:
Good point. Removed in https://github.com/plotly/plotly.js/pull/5240/commits/08aa30098aa7efdea0205ad221a6c654e3fc4ff6.